src/languages/es.ts

```diff
@@ -775,6 +775,8 @@ export default {
         enterDefaultContact: 'Tu método de contacto predeterminado',
         defaultContact: 'Método de contacto predeterminado:',
         enterYourDefaultContactMethod: 'Por favor, introduce tu método de contacto predeterminado para cerrar tu cuenta.',
+        hasSharedPolicies:
+            'Sigues siendo propietario de espacios de trabajo compartidos con otros usuarios. Elimina el espacio de trabajo o asigna otro propietario antes de volver a intentarlo.',
```

Comment:
For reference: https://github.com/Expensify/App/issues/27553#issuecomment-1768641981